### PR TITLE
Updated Russian ML translations

### PR DESCRIPTION
**Pull request by juicejuicejuice**
- Changed some phrases
- Fixed grammar errors I found

### CODE REVIEW

File: plugins/lang/mapsmenu.txt [hunk 1]
```diff
@@ -517,7 +517,7 @@ SEL_MAPS = Выбранные карты
 ALREADY_VOT = Голосование уже начато...
 NO_MAPS_MENU = В меню нет карт
 VOT_NOW_ALLOW = В данный момент голосование запрещено
-WHICH_MAP = Какую карту вы бы хотели?
+WHICH_MAP = Какую карту Вы бы хотели?
```
Comment (WPMGPRoSToTeMa) [inline plugins/lang/mapsmenu.txt]: It is not required.

**Comment by OciXCrom** [comment]
The TRANSF_SILENT translation you added is for Bosnian language, not Macedonian, and it's incorrectly translated.

**Comment by juicejuicejuice** [comment]
> The TRANSF_SILENT translation you added is for Bosnian language, not Macedonian, and it's incorrectly translated.

Thanks for pointing that out.

**Comment by Arkshine** [comment]
@juicejuicejuice Are you finished with the modifications?

**Comment by OciXCrom** [comment]
As @WPMGPRoSToTeMa said, it would be better to revert "Вы" to "вы" for consistency with other languages.

Вы = You
вы = you

"You" with a capital letter isn't used anywhere, there's no need to be that formal.

File: plugins/lang/mapsmenu.txt (hunk 1 above)
Comment (WPMGPRoSToTeMa) [reply plugins/lang/mapsmenu.txt]: Maybe we can translate it as: "Голосование за следующую карту" or "Проголосуйте за следующую карту"?

**Comment by juicejuicejuice** [comment]
> @juicejuicejuice Are you finished with the modifications?

Finished just now!

**Pull request merged**